.github/config.yml

```diff
@@ -0,0 +1,29 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+# Heavily inspired by https://github.com/electron/electron/blob/master/.github/config.yml
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
+  To help make it easier for us to investigate your issue, please follow the [contributing guidelines](https://github.com/videojs/video.js/blob/master/CONTRIBUTING.md) and [issue template](https://github.com/videojs/video.js/blob/master/.github/ISSUE_TEMPLATE.md).
+
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: |
+  💖 Thanks for opening this pull request! 💖
+  Things that will help get your PR across the finish line:
+
+  - Run `npm run lint -- --errors` locally to catch formatting errors earlier.
+  - Include tests when adding/changing behavior.
+  - Include screenshots and animated GIFs whenever possible.
```

Comment:
Do they need to do this? Won't netlify do this for us most of the time? Should we mention that here somehow?

Reply:
I think it makes sense, particularly for UI changes. Netlify gives us a live example but we still can't be sure what the intent and broken behavior was without it.
Also, screenshots and gifs can help debug stuff that aren't directly UI changes but rendering issues and what not.

Reply:
OK, yeah your right.